components/widgets/links/upgrade_link.tsx

```diff
@@ -22,7 +22,7 @@ export interface UpgradeLinkProps {
     styleLink?: boolean; // show as a anchor link
 }
 
-const UpgradeLink: React.FC<UpgradeLinkProps> = (props: UpgradeLinkProps) => {
+const UpgradeLink = (props: UpgradeLinkProps) => {
```

Comment:
this is not related to the PR, but is part of the effort to clean up components that I see using React.FC unnecessarily.